### DISCUSSION TITLE
It tracks as online/offline instead of greying out as disabled or shut down.

### DISCUSSION THREAD

**Discussion by Zulkiers** [InternetConnectionMonitor/icm-public#100] [Ideas and Feature requests]
I think it should make blank space when you disable the ICM or shut down your browser/PC.
Not track as the last known, being online or offline.
Sometimes I will shut down while online, and then lose my internet for a few hours. It tracks as online during that shut down time.
Sometimes I will shut down during a glitch or bad internet condition, classified as offline, and then have good internet a couple hours later and for a while after that, but it classifies it as offline.
I feel it should instead have blank space for the times it's not active, for the times it can't track.
Then it would have more reliable data.

**Comment by dmishh** [comment]
@Zulkiers makes sense, but it will require some rework in the way downtime is calculated. It's not an easy thing, but definitely a good idea. I'm also changing to a feature since there is a request for a new status, not an actual bug in the ICM

**Comment by dmishh** [comment]
This idea should cover the case for ICM being off: https://github.com/InternetConnectionMonitor/icm-public/discussions/104#discussioncomment-10170559

**Comment by Zulkiers** [comment]
Update: I had another thought.
You know how you calculate "downtime, today" and such?
Add a second tracker "Uptime, today"
When the PC (and ICM) is on, it will track both uptime and downtime, and show availability based off those, rather than global time.
Time where neither uptime nor downtime are tracked would be the offline time or disabled time, the invalid time, excluded time, because it didn't track anything.
This way we have more accurate readings of our internet in proportion to how we use it, when we are on.

What do you think @dmishh ?

**Reply by dmishh** [reply]
@Zulkiers yeah, that's another way of doing this. I'll keep it in mind. Thanks!

**Comment by dmishh** [comment]
Duplicate of #104
